make/CMake_events.cmake

```diff
@@ -46,7 +46,9 @@ add_custom_command(
        ${SYSLOG_SRCS}
   VERBATIM # recommended: p260
   )
-if (WIN32)
+if (WIN32 AND NOT ("${CMAKE_GENERATOR}" MATCHES "MSYS Makefiles"))
+  # In MSYS2 we not define CMAKE_GENERATOR
+  # so we can't use Windows event log.
   set(EVENTS_SRCS ${PROJECT_BINARY_DIR}/events.h)
```

Comment:
(events.h is included in many files, so there will be build failures.  But I assume you will address those separately.)

Reply:
@derekbruening Yes, sure.